edunext_openedx_extensions/manage_api/views.py

```diff
@@ -126,6 +129,51 @@ def post(self, request):
 
         return JsonResponse({"success": True}, status=201)
 
+    def get(self, request, **kwargs):  # pylint: disable=unused-argument
+        """
+        TODO: add me
+        """
+
+        username = request.GET.get('username')
+        email = request.GET.get('email')
+
+        result, status = self._get_user(username, email)
+
+        return JsonResponse(result, status=status)
+
+    def _get_user(self, username, email):
+        """
+        Helper method to return the username and email of an existing user.
+        """
+        try:
+            existing_user = User.objects.get(username=username)
```

Comment:
Yes, the answer is no. I'm planning to use this API endpoint to decide if the user exists or not. I do not fire any requests to user-api at edxapp.